sdk/schema/executable.go

```diff
@@ -23,6 +23,9 @@ type Executable struct {
 
 	// (Optional) Whether the exectuable needs authentication for certain args.
 	NeedsAuth sdk.NeedsAuthentication
+
+	// (Optional) words to prompt about a credential
+	AuthPrompt string
```

Comment:
I am failing to see what this does. Could you please elaborate on why this is needed in this scenario?

Reply:
Does the new comment clarify enough?

Reply:
To not create confusion with the biometric auth prompt functionality, how about:
```suggestion
	Description string
```